library/network_connections.py

```diff
@@ -173,7 +173,7 @@ def _link_read_permaddress(ifname):
     @staticmethod
     def _link_infos_fetch():
         links = {}
-        for ifname in os.listdir("/sys/class/net/"):
+        for ifname in sorted(os.listdir("/sys/class/net/")):
```

Comment:
Please reduce the commit message to the essence of why  this change is useful.

Reply:
I rephrased it to capture the essence of the reason for the change.